### ISSUE TITLE
NVDA screen reader doesn't read label of any checkboxes

### ISSUE THREAD

**Issue by madryk** [CeON/fairchive#896]

**Comment by madryk** [comment]
Problem lays in primefaces core.css - NVDA don't read the checkbox because it doesn't occupy any space in html (`scale(0)` - rule)
